presto-hive/src/test/java/io/prestosql/plugin/hive/HiveTestUtils.java

```diff
@@ -156,4 +157,20 @@ public static Slice longDecimal(String value)
     {
         return encodeScaledValue(new BigDecimal(value));
     }
+
+    public static MethodHandle distinctFromOperator(Type type)
+    {
+        Signature signature = FUNCTION_REGISTRY.resolveOperator(IS_DISTINCT_FROM, ImmutableList.of(type, type));
```

Comment:
why did you use `is_distinct_from` and not `is_equal_to`? Is it due `Maps need to be compared without regard to order.`? Maps are equal if they have same entries and they are sorted in the same way?

Reply:
Equality for a map with null values will return null. I'm not sure if this is correct, but that's how it works today.